src/TestFramework/Extension.Shared/TestContext.cs

```diff
@@ -21,6 +22,11 @@ public abstract class TestContext
         /// </summary>
         public abstract IDictionary Properties { get; }
 
+        /// <summary>
+        /// Gets or sets the cancellation token source that is cancelled when test timesout or when cancelled the test will be aborted
```

Comment:
Nit: Gets or sets the cancellation token source. It is cancelled when the test times out. When cancelled, the test gets aborted.

Reply:
Done.